.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,4 @@
 - [ ] closes #xxxx
 - [ ] tests added / passed
-- [ ] passes `black pandas`
-- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
+- [ ] passes `./scripts/quick_style_check.sh`
```

Comment:
Will people developing on Windows be able to run this?

Reply:
That's in part why I'm pinging the whole group about the direction.

If we want to use a script like I have now, I would rewrite this in Python so that it's cross-platform.  As of now, it would only work on Linux distributions (not even MacOS because its implementation of `xargs` does not recognize `-r` since it doesn't need it)